docs/notebooks/environment.yml

```diff
@@ -5,13 +5,14 @@ channels:
 dependencies:
  - python==3.8
  - numpy>=1.14.0
- - pandas>=1.1.2
- - anndata==0.7.8
+ - pandas>=1.5.3
+ - anndata==0.8.0
  - numba>=0.53.0
  - scanpy>=1.6.0
  - jupyterlab>=3
  - zarr>=2.5.0
  - boto3>=1.16.30
+ - starlette==0.30.0
```

Comment:
this dependency is needed when we call `anndata_wrapper_inst.auto_view_config(vc)`, part of the `convert_cellbrowser_project_to_vitessce_config` function. I upgraded straight to 0.30, instead of 0.14 (as it is in the `vitessce-python-dev` environment) to avoid having to install `'aiofiles>=0.6.0'`

Reply:
I think we need to address this first https://github.com/vitessce/vitessce-python/issues/254